extensions/wikia/MiniEditor/js/MiniEditor.js

```diff
@@ -234,6 +234,26 @@ var MiniEditor = {
 				startupFocus: !preloading
 			}, options.config), event);
 
+			// VOLDEV-25: Provide LinkSuggest support if necessary
+			if (window.wgEnableMiniEditorLinkSuggest) {
+				if (this.config.editorSuite === 'ckeditor') {
+					// RTE
+					wikiaEditor.on('ck-mode', function () {
+						if (wikiaEditor.ck.mode !== 'wysiwyg') {
```

Comment:
It would be better to check if the current mode is the desired mode, so something like: ```wikiaEditor.ck.mode === 'wikitext'``` (I don't know if it's called 'wikitext' I'm just making that up).